tzrec/tools/create_fg_json.py

```diff
@@ -137,6 +137,11 @@
                     resource = o.create_resource(
                         fname, "file", file_obj=open(fpath, "rb")
                     )
+                else:
+                    raise ValueError(
+                        f"{fname} has already existed in project:{project}, "
+                        f"you can add '--force_update_resource true' in command"
```

Comment:
{fname} already existed in the {project}. You can add '--force_update_resource' in the command to force update.